Dockerfile

```diff
@@ -57,7 +57,6 @@ RUN echo "Starting the build ..." &&\
 ############################################################
 
 # Volumes
-VOLUME ["/etc/ssl/certs"]
```

Comment:
Hey, many thanks for the contribution.

You'r right, it is still needed. Thanks for the notice!